### PR TITLE
Add issue templates

### PR DESCRIPTION
**Pull request by othon2000**
linked to https://github.com/zalando-nakadi/fahrschein/pull/265/files

### CODE REVIEW

**Comment by othon2000** [comment]
:+1:

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Logs**
+If applicable, add logs to help demonstrate your problem.
+
+**Environment:**
+ - OS: [e.g. Ubuntu, OS X, Windows]
```
Comment (otrosien) [inline .github/ISSUE_TEMPLATE/bug_report.md:27]: do we really need OS?

Comment (otrosien) [reply .github/ISSUE_TEMPLATE/bug_report.md:27]: java commandline options would be more interesting maybe.

**Comment by othon2000** [comment]
:+1:

**Comment by othon2000** [comment]
:+1:

**Comment by otrosien** [comment]
👍

**Pull request merged**